src/pages/AboutPage/AboutPage.jsx

```diff
@@ -0,0 +1,25 @@
+// import { Link } from "react-router-dom";
+
+import './AboutPage.css';
+
+function AboutPage() {
+
+    return (
+        <section className="section about">
+            <div className="photo">
+                <img src="/mentoring1.jpeg" alt="" />
+            </div>
+            <div className='about-text'>
+                <h1>About US</h1>
```

Comment:
Is US supposed to be in caps?

Reply:
No the US in capitals is an accident, will change it now.